File: 2037-count-square-sum-triples/count-square-sum-triples.py
class Solution(object):
    def countTriples(self, n):
        """
        :type n: int
        :rtype: int
        """
        count=0
        for i in range(1,n+1):
            for j in range(1,n+1):
                cs=i*i+j*j
                c=int(cs**0.5)
                if(c*c==cs and c<=n):
                    count+=1

        return count
        

File: 2037-count-square-sum-triples/count-square-sum-triples.java
class Solution {
    public int countTriples(int n) {
        int count =0 ;
        for(int i=1;i<=n;i++){
            for(int j=1;j<=n;j++){
                int cs=i*i+j*j;
                int c=(int) Math.sqrt(cs);
                if(c*c==cs && c<=n) count++;
            }
        }
        return count;
    }
}